cernopendata_client/cli.py

```diff
@@ -68,6 +68,60 @@ def version():
     display_message(msg=__version__)
 
 
+def filter_metadata(output_field, filters, output_json):
+    # noqa: D301
+    """Filter metadata objects based on specified criteria.
+
+    :param output_field: Name of the array containing objects to access
+    :param filters: Argument of the --filter option in the format field_name=value
```

Comment:
`field_name=value` could be misunderstood: is value supposed to be the field name? Or the value of the field?  Therefore it would be better to keep "somefield=somevalue" speaking abstractly, or say "some_field_name=some_field_value" to be more precise.